tests/network/vmi_multus.go

```diff
@@ -332,26 +353,55 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		})
 
 		Context("VirtualMachineInstance with Linux bridge plugin interface", func() {
-			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName string) {
+			getInterfaceAddressByNetworkName := func(vmi *v1.VirtualMachineInstance, networkName string) (string, error) {
+				vmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
+
+				ifaceAddress := ""
+				for _, iface := range vmi.Status.Interfaces {
+					if iface.Name == networkName {
+						ifaceAddress = iface.IP
+					}
+				}
+				return ifaceAddress, err
+			}
+
+			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName, staticIPVm1, staticIPVm2 string) {
+				if staticIPVm2 == "" || staticIPVm1 == "" {
+					ipam := "\\\"type\\\": \\\"host-local\\\", \\\"subnet\\\": \\\"10.1.1.0/24\\\""
+					Expect(createBridgeNetworkAttachementDefinition("linux-bridge-net-ipam", "0", ipam))
+				}
+
 				vmiOne := createVMIOnNode(interfaces, networks)
 				vmiTwo := createVMIOnNode(interfaces, networks)
 
 				tests.WaitUntilVMIReady(vmiOne, console.LoginToAlpine)
 				tests.WaitUntilVMIReady(vmiTwo, console.LoginToAlpine)
 
-				Expect(configInterface(vmiOne, ifaceName, "10.1.1.1/24")).To(Succeed())
+				Expect(configureAlpineInterface(vmiOne, ifaceName, staticIPVm1)).To(Succeed())
 				By(fmt.Sprintf("checking virtual machine interface %s state", ifaceName))
 				Expect(checkInterface(vmiOne, ifaceName)).To(Succeed())
 
-				Expect(configInterface(vmiTwo, ifaceName, "10.1.1.2/24")).To(Succeed())
+				Expect(configureAlpineInterface(vmiTwo, ifaceName, staticIPVm2)).To(Succeed())
 				By(fmt.Sprintf("checking virtual machine interface %s state", ifaceName))
 				Expect(checkInterface(vmiTwo, ifaceName)).To(Succeed())
 
+				ipAddr := ""
+				if staticIPVm2 == "" {
+					By("will ping secondary interface")
+					ipAddr, err = getInterfaceAddressByNetworkName(vmiTwo, networks[1].Name)
```

Comment:
You depend here of a network slice w/ 2 items or more. I miss that assertion somewhere (while I do realize that it's an input of your test).